lib/SILGen/SILGenFunction.h

```diff
@@ -945,6 +945,8 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
 
   void emitConditionalPBD(PatternBindingDecl *PBD, SILBasicBlock *FailBB);
 
+  void usingImplicitVariablesForPattern(Pattern *pattern, CaseStmt *stmt,
+                                        const std::function<void(void)> &f);
```

Comment:
Nitpick: we have llvm::function_ref for generalized functions that don't need to be copied to the heap.